docs/iris/src/further_topics/metadata.rst

```diff
@@ -0,0 +1,987 @@
+.. _metadata:
+
+********
+Metadata
+********
+
+This section provides a detailed overview of how your metadata is managed
+within Iris. In particular, it discusses what metadata is, where it fits
+into Iris, and more importantly how you can create, access, manipulate,
+and analyse your metadata.
+
+All the finer details covered here may not be entirely relevant to your use
+case, but it's here if you ever need it. In fact, you may want to skip
+straight ahead to :ref:`richer metadata`, and take it from there.
+
+
+Introduction
```

Comment:
As I mentioned in my main comment:

It took me a while to follow the introduction and first section as I think the paragraphs hop around slightly.  All the vital information in there, so I doubt anyone else will notice, but I would:
* Introduction:  Move the first paragraph down, so all the class information is together, and give a more explicit reference to where the Cube, DimCoord, AuxCoord info is introduced wrt CF by re-referencing the "Data structures doc"

```suggestion
Introduction
============

As discussed in :ref:`iris_data_structures`, Iris draws heavily from the
`NetCDF CF Metadata Conventions`_ as a source for its data model, thus building
on the widely recognised and understood terminology defined within those
`CF Conventions`_ by the scientific community.

In :ref:`iris_data_structures` we also introduced several classes in Iris that care about your ``data``,
and also your ``metadata``: the :class:`~iris.cube.Cube`, the :class:`~iris.coords.AuxCoord`, and the
:class:`~iris.coords.DimCoord`.  In addition to these Iris models several other classes of `CF Conventions`_ metadata: the :class:`~iris.coords.AncillaryVariable`
(see `Ancillary Data`_ and `Flags`_), the :class:`~iris.coords.CellMeasure`
(see `Cell Measures`_), and also the :class:`~iris.aux_factory.AuxCoordFactory`
(see `Parametric Vertical Coordinate`_).

```